dns/icon_dns/connection/connection.py

```diff
@@ -8,7 +9,9 @@ def __init__(self):
         super(self.__class__, self).__init__(input=ConnectionSchema())
 
     def connect(self, params):
+        self.logger.info("Connect: Connecting...")
```

Comment:
Can you remove this line? There's no connection happening so we don't need to log it.